dataland-framework-toolbox/src/main/kotlin/org/dataland/frameworktoolbox/frameworks/esgquestionnaire/EsgQuestionnaireFramework.kt

```diff
@@ -164,7 +164,7 @@ class EsgQuestionnaireFramework : PavedRoadFramework(
 
     private fun setEuTaxonomieKompassAktivitaeten(component: MultiSelectComponent) {
         component.label = "EU Taxonomie Kompass Aktivitäten"
-        component.explanation = "Welche Aktivitäten gem. dem EU Taxonomie-Kompass übt das Unternehmen aus?"
+        component.uploadPageExplanation = "Welche Aktivitäten gem. dem EU Taxonomie-Kompass übt das Unternehmen aus?"
```

Comment:
Ich bin mir nicht ganz sicher wieso sich das hier geändert hat, aber müsste es jetzt nicht möglich sein in GDV auch die verschiedenen Tooltips auf View- und Upload-page zu zeigen? Hätte gedacht das passiert automatisch, scheint aber aktuell nicht so zu sein.